src/trumbowyg.js

```diff
@@ -480,16 +480,10 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
 
           if(!protocol) { return; }
 
-          if(typeof(protocol) === 'boolean') {
-              return 'https://'
-          } else if(typeof(protocol) === 'string') {
-              if(/^https?:\/\/$/.test(protocol)) { return protocol; }
-
-              if(/^https?$/.test(protocol)) {
-                  return protocol + '://'
-              }
-          }
-        }
+          if(typeof(protocol) !== 'string') { return 'https://'; }
+          if(/^https?:\/\/$/.test(protocol)) { return protocol; }
+          if(/^https?$/.test(protocol)) { return protocol + '://'; }
```

Comment:
That's a lot of code to just get `https://` or `http://` :/

Reply:
What about others protocols? If I want put some ftp:// or whatever?

Reply:
Thanks @Alex-D. 

Would you prefer to allow any string value in the option, and append `://` if it's missing?

So:

- a `true` value defaults to `https://`
- a string without the correct suffix has `://` appended to it
- a string with the correct suffix is prepended as is

Reply:
yep I think it could be cool :)